src/System.ComponentModel.TypeConverter/src/System/ComponentModel/MaskedTextProvider.cs

```diff
@@ -668,47 +625,23 @@ public bool IsPassword
         /// <summary>
         ///     A negative value representing an index outside the test string.
         /// </summary>
-        public static int InvalidIndex
-        {
-            get
-            {
-                return invalidIndex;
-            }
-        }
+        public static int InvalidIndex => invalidIndex;
```

Comment:
Should we update all constants from this class to be uppercase? e.g `invalidIndex` to `INVALID_INDEX` ?